src/frontends/pytorch/src/transforms/prim_list_construct_pad.cpp

```diff
@@ -61,19 +61,45 @@ PrimListConstructPadReplacer::PrimListConstructPadReplacer() {
     // transformation for case aten::pad + prim::ListConstruct as paddings
     auto pad_op = ov::pass::pattern::wrap_type<ov::op::util::FrameworkNode>();
     ov::matcher_pass_callback callback = [](ov::pass::pattern::Matcher& m) {
-        auto pad_op = cast_fw_node(m.get_match_root(), "aten::pad");
-        if (!pad_op) {
+        Output<Node> input_node;
+        Output<Node> padding;
+        Output<Node> pad_value;
+        std::string mode;
+        NodeVector rt_info = {};
+        std::shared_ptr<ov::op::util::FrameworkNode> pad_op;
+        if ((pad_op = cast_fw_node(m.get_match_root(), "aten::pad"))) {
+            mode = "constant";
+            input_node = pad_op->input_value(0);
+            padding = pad_op->input_value(1);
+            auto mode_node = pad_op->input_value(2).get_node_shared_ptr();
+            pad_value = pad_op->input_value(3);
+            rt_info = {pad_op,
+                       input_node.get_node_shared_ptr(),
+                       padding.get_node_shared_ptr(),
+                       mode_node,
+                       pad_value.get_node_shared_ptr()};
+            if (const auto& fw_node_mode = cast_fw_node(mode_node, "prim::Constant")) {
+                const auto& attrs = fw_node_mode->get_attrs();
+                if (attrs.find("string_value") != attrs.end()) {
+                    mode = attrs.at("string_value");
+                }
+            }
+        } else if ((pad_op = cast_fw_node(m.get_match_root(), "aten::reflection_pad2d"))) {
+            mode = "reflect";
+            input_node = pad_op->input_value(0);
+            padding = pad_op->input_value(1);
+            // Pad value is used only for constant pad, fill with 0 as placeholder.
+            pad_value = v0::Constant::create(element::f32, Shape{}, {0});
+            rt_info = {pad_op, input_node.get_node_shared_ptr(), padding.get_node_shared_ptr()};
```

Comment:
Why do you create this NodeVector? I can't it being used anywhere. In fact if you meant to use it to copy rt info it is not needed, because all created operation are handled by NodeRegistry here.

Reply:
Thanks, deleted. I've added it before changes with NodeRegistry were introduced and forgot to delete it while solving merge conflicts